ui/raidboss/data/04-sb/raid/o2s.js

```diff
@@ -244,29 +292,37 @@ export default {
         },
       },
     },
+    {
+      id: 'O2S Maniacal Probe You',
+      netRegex: NetRegexes.headMarker({ id: '0005 ' }),
+      condition: Conditions.targetIsYou(),
+      alertText: (_data, _matches, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Probe on YOU',
+        },
+      },
+    },
     {
       id: 'O2S Unstable Gravity',
       netRegex: NetRegexes.gainsEffect({ effectId: '550' }),
       condition: Conditions.targetIsYou(),
-      delaySeconds: 9,
-      alarmText: (_data, _matches, output) => output.elevateOutsideStack(),
-      tts: (_data, _matches, output) => output.floatForBomb(),
+      alarmText: (_data, _matches, output) => output.text(),
       outputStrings: {
-        elevateOutsideStack: {
-          en: 'Unstable Gravity: Elevate and outside stack',
-          de: 'Schwerkraftschwankung: Schweben und außen stacken',
-          fr: 'Gravité instable : Lévitez et packez-vous à l\'extérieur',
-          ja: 'グラビティバースト: 浮上集合',
-          cn: '升起并分摊',
-          ko: '중력 폭탄: 공중부양 및 쉐어',
+        text: {
+          en: 'Elevate (Unstable Gravity)',
         },
-        floatForBomb: {
-          en: 'float for bomb',
-          de: 'schweben für bombe',
-          fr: 'Flottez pour la bombe',
-          ja: '浮上集合',
-          cn: '升起并分摊',
-          ko: '중력 폭탄',
+      },
+    },
+    {
+      id: 'O2S Unstable Gravity Delayed',
+      netRegex: NetRegexes.gainsEffect({ effectId: '550' }),
+      condition: Conditions.targetIsYou(),
+      delaySeconds: 9,
+      alertText: (_data, _matches, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Move Slightly Outside Stack',
```

Comment:
```suggestion
          en: 'Move Slightly Outside Stack',
          ko: '쉐어 범위에서 살짝 멀어지기',
```

Reply:
You get knocked regardless if you're in the stack or not, right? Being slightly askew makes it easier to point your knockback direction? Do we have other callouts similar to "Get Knocked Back"?

Reply:
Yes you get knocked back regardless.  Arguably the earlier "elevate" call covers "get knocked back".  I mostly just wanted a "this is the time where the big circles in the arena disappear and you can walk a step in the direction to make sure you get pushed in the right direction" sort of call.  Do you have a better way to word this, or should I skip this?

Reply:
Looking around, we have some "Get Knocked to X" [[1]](https://github.com/quisquous/cactbot/blob/067c856c754bbde68eb5fb97be0ba217d3557701/ui/raidboss/data/05-shb/dungeon/matoyas_relict.js#L147), [[2]](https://github.com/quisquous/cactbot/blob/14ea3da14baac3f31e48222d47c1d774416e6624/ui/raidboss/data/05-shb/eureka/delubrum_reginae.js#L727), [[3]](https://github.com/quisquous/cactbot/blob/1fa332b73c7bd0d2fcf1c7f8c1f7b9cc99f0a043/ui/raidboss/data/05-shb/raid/e7s.js#L486) callouts. Does a "Get Knocked to Edge" say the same thing here? I feel like neither really says what the mechanic does if you were running this for the first time (although they might save you from a wipe?)

Reply:
Hmm.  Agreed that there are other similar callouts, but I think the stack here makes this one different? Agreed that none of this would save you, but I wanted to differentiate "get knocked back" where somebody might just run to the edge vs "move slightly" where you are still in the stack and get knocked back? Maybe I'm overthinking it.

Reply:
I feel like "Get Knocked to Edge" has a lower likelihood of someone standing outside of the stack than "Move Slightly Outside Stack."

That being said, I really don't have strong opinions here.